packages/amplify-e2e-tests/src/plugin/index.ts

```diff
@@ -1,34 +1,31 @@
-export * from './new-plugin';
+export * from './newPlugin';
 export * from './verifyPluginStructure';
+export * from './pluginConfigure';
 
 import { nspawn as spawn } from 'amplify-e2e-core';
 import { getCLIPath } from '../utils';
 
 export function help(cwd: string) {
   return new Promise((resolve, reject) => {
-    spawn(getCLIPath(), ['plugin', 'help'], { cwd, stripColors: true })
-      .wait(/.*/)
```

Comment:
Why remove the ```.wait(/.*/)```?

Shouldn't we be checking that the help command outputs something?

Reply:
We are not waiting for any  particular output to respond to. 
other example:
https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-e2e-tests/src/categories/codegen.ts#L6